beta-src/src/state/game/game-api-slice.ts

```diff
@@ -150,7 +150,7 @@ const resetOrder = (state) => {
   } = current(state);
   if (type !== "hold") {
     const command: GameCommand = {
-      command: "NONE",
+      command: type === "Retreat" ? "DISLODGED" : "NONE",
```

Comment:
type here should be an `OrderType` which is in `/src/types/state/OrderType`. We should keep them all lowercase for consistency.